packages/zk-certificates/tasks/reissueZkCertificate.ts

```diff
@@ -108,12 +143,52 @@ async function main(args: any, hre: HardhatRuntimeEnvironment) {
     ),
   );
 
+  console.log('Register zkCertificate to the queue to readd...');
+  [startTime, expirationTime] = await registerZkCert(
+    newZkCertificate.leafHash,
+    recordRegistry,
+    issuer,
+  );
+
+  currentBlock = await provider.getBlockNumber();
+  lastBlockTime = (await provider.getBlock(currentBlock)).timestamp;
+
+  // wait until start time
+  while (lastBlockTime < startTime) {
```

Comment:
Looks like the code for waiting is the same in all three scripts. I would put it as a function into the registryTools. Then we have one function instead of four copy pasted versions.